applicationinsights-react-js/package.json

```diff
@@ -68,7 +68,8 @@
         "@microsoft/applicationinsights-shims": "^3.0.1",
         "@microsoft/applicationinsights-core-js": "^3.0.3",
         "@microsoft/applicationinsights-common": "^3.0.3",
-        "@microsoft/dynamicproto-js": "^2.0.2"
+        "@microsoft/dynamicproto-js": "^2.0.2",
+        "@nevware21/ts-utils": "^0.10.1"
```

Comment:
This should use the same range as "@microsoft/applicationinsights-core-js" which is
`"@nevware21/ts-utils": ">= 0.10.1 < 2.x"`

So that we won't have to release an updated version once v1.x is released and a later version of `@microsoft/applicationinsights-core-js` within the same range (eg 3.0.x) updates to use it as that will break all consumers or cause multiple versions to be included.